src/sonic-dhcp-server/dhcp_server/dhcpservd/dhcpservd.py

```diff
@@ -7,21 +7,35 @@
 from .dhcp_cfggen import DhcpServCfgGenerator
 from .dhcp_lease import LeaseManager
 from dhcp_server.common.utils import DhcpDbConnector
+from dhcp_server.common.dhcp_db_monitor import DhcpServdDbMonitor
+from swsscommon import swsscommon
 
 KEA_DHCP4_CONFIG = "/etc/kea/kea-dhcp4.conf"
 KEA_DHCP4_PROC_NAME = "kea-dhcp4"
 KEA_LEASE_FILE_PATH = "/tmp/kea-lease.csv"
 REDIS_SOCK_PATH = "/var/run/redis/redis.sock"
+DHCP_SERVER_IPV4 = "DHCP_SERVER_IPV4"
+DHCP_SERVER_IPV4_PORT = "DHCP_SERVER_IPV4_PORT"
+DHCP_SERVER_IPV4_RANGE = "DHCP_SERVER_IPV4_RANGE"
+VLAN = "VLAN"
+VLAN_MEMBER = "VLAN_MEMBER"
+VLAN_INTERFACE = "VLAN_INTERFACE"
+DHCP_SERVER_IPV4_CUSTOMIZED_OPTIONS = "DHCP_SERVER_IPV4_CUSTOMIZED_OPTIONS"
 DHCP_SERVER_IPV4_SERVER_IP = "DHCP_SERVER_IPV4_SERVER_IP"
 DHCP_SERVER_INTERFACE = "eth0"
 AF_INET = 2
+DEFAULT_SELECT_TIMEOUT = 5000  # millisecond
 
 
 class DhcpServd(object):
-    def __init__(self, dhcp_cfg_generator, db_connector, kea_dhcp4_config_path=KEA_DHCP4_CONFIG):
+    sel = None
+
+    def __init__(self, dhcp_cfg_generator, db_connector, kea_dhcp4_config_path=KEA_DHCP4_CONFIG,
+                 select_timeout=DEFAULT_SELECT_TIMEOUT):
         self.dhcp_cfg_generator = dhcp_cfg_generator
         self.db_connector = db_connector
         self.kea_dhcp4_config_path = kea_dhcp4_config_path
+        self.dhcp_servd_monitor = DhcpServdDbMonitor(db_connector, select_timeout)
```

Comment:
inject this object from parameter just like db_connector

Reply:
Fixed